client.py

```diff
@@ -0,0 +1,74 @@
+import tkinter as tk
+import requests
+
+
+def main():
+    """
+    run interactive tkinter envinroment as client
+    """
+    root = tk.Tk()
+    canvas1 = tk.Canvas(root, width=800, height=300)
+    canvas1.pack()
+    global label3
+    label3 = tk.Label(root, text="")
+    label3.config(font=('helvetice', 14))
+    label3.pack(side="left")
+    entry1 = tk.Entry(root, width=80)
+    canvas1.create_window(400, 200, window=entry1)
+    button1 = tk.Button(text="run command",
+                        command=lambda: send_command(entry1, canvas1, root))
+    canvas1.create_window(400, 250, window=button1)
+
+    label1 = tk.Label(root, text="run a command")
+    label1.config(font=('helvetica', 30))
+    canvas1.create_window(400, 60, window=label1)
+
+    label2 = tk.Label(root, text="type the command here:")
+    label2.config(font=('helvetica', 22))
+    canvas1.create_window(400, 150, window=label2)
+    root.mainloop()
```

Comment:
Too big of a functionality for `main` that only needs to call the relevant functions that run the program. You are configuring the `tkinter` here, so move the functionality to a function `configure_window` or something.